src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -222,10 +223,15 @@ HttpStatusCode mongoSubscribeContext
 
     LM_T(LmtSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
 
+    // FIXME Px: In the near future, the subscription carries the HttpInfo
```

Comment:
Not exactly... what I mean is that after the refactoring we are doing righ now to create independent mongoCreateSubscription() and mongoUpdateSubscription() methods for NGSIv2 all the `if (apiVersion == "v2")` is no longer needed.

Thus, mongoSubscribreContext() will be used only in the NGSIv1 case and, under that circustances, you only need 

```
httpInfo.url = requestP->reference.get();
```

now and forever :)